lnd_test.go

```diff
@@ -9415,6 +9544,8 @@ func testMultiHopRemoteForceCloseOnChainHtlcTimeout(net *lntest.NetworkHarness,
 		// registration. As a result, we'll mine another block and
 		// repeat the check. If it doesn't go through this time, then
 		// we'll fail.
+		// TODO(halseth): use waitForChannelPendingForceClose to avoid
```

Comment:
Any reason we can't use it here yet?

Reply:
I _thiiink_ we can, but haven't really looked into it. Clarified the todo.